internal/app/api/api.go

```diff
@@ -404,6 +335,56 @@ func (a *Client) DrainFeed(ctx context.Context, request *GetFeedRequest, feedFn
 	return nil
 }
 
+// GetDeletedInspections returns response from AccountsActivityLog or error
+func (a *Client) GetDeletedInspections(ctx context.Context, request *GetAccountsActivityLogRequest) (*GetAccountsActivityLogResponse, error) {
```

Comment:
This this really `GetDeletedInspections`? Or is it it `ListOrganisationActivityLog`?

Reply:
good point. Was initially Deleted, but after refactor the request obj ... forgot about it.